convert2rhel/unit_tests/checks_test.py

```diff
@@ -334,17 +431,28 @@ def test_c2r_up_to_date_multiple_packages(self, caplog, convert2rhel_latest_vers
         with pytest.raises(SystemExit):
             checks.check_convert2rhel_latest()
 
+        local_version, package_version = convert2rhel_latest_version_test
+        print(len(package_version))
+        if len(package_version) > 110:
+            package_version = package_version[97:103]
+        else:
+            package_version = package_version[93:97]
+        print(package_version)
```

Comment:
Debug prints here should be removed
```suggestion

        if len(package_version) > 110:
            package_version = package_version[97:103]
        else:
            package_version = package_version[93:97]

```